### PR TITLE
Update Ed25519Signature2018 to use jsigs v9, add context, etc.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Depends on: PR https://github.com/digitalbazaar/jws-linked-data-signature/pull/2

### CODE REVIEW

File: lib/Ed25519Signature2018.js [hunk 1]
```diff
@@ -7,31 +7,46 @@ import {
...
-   * @param [useNativeCanonize] {boolean} true to use a native canonize
-   *   algorithm.
+   * @param {string|Date} [options.date] - Signing date to use if not passed.
+   * @param {boolean} [options.useNativeCanonize] - Whether to use a native
+   *   canonize algorithm.
    */
-  constructor({
-    signer, key, verificationMethod, proof, date, useNativeCanonize} = {}) {
+  constructor({key, signer, verifier, proof, date, useNativeCanonize} = {}) {
     super({type: 'Ed25519Signature2018', alg: 'EdDSA',
```
Comment (dlongley) [inline lib/Ed25519Signature2018.js:43]:
```suggestion
    super({
      type: 'Ed25519Signature2018', alg: 'EdDSA',
```

**Review by dlongley — APPROVED** [review]
AFAICT, this is using the same pattern as other suites -- nothing new right?

File: lib/Ed25519Signature2018.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/Ed25519Signature2018.js]: Fixed.

**Comment by dmitrizagidulin** [comment]
@dlongley 
> AFAICT, this is using the same pattern as other suites -- nothing new right?

Yes, though I did just move the "deal with `credentials/v1` conflict" logic to this PR.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**